thirdeye/thirdeye-frontend/app/pods/components/anomaly-id/template.hbs

```diff
@@ -0,0 +1,13 @@
+<li class="anomaly-id__item anomaly-id__item--title">
```

Comment:
Not using --metric, but yes using --value. The scss needed a bit of re-arranging.